explainerdashboard/explainers.py

```diff
@@ -50,6 +50,7 @@
 
 
 from .explainer_methods import *
+from .explainer_methods import check_dtype_of
```

Comment:
you can add `check_dtype_of` to the` __all__` at the start in explainer_methods.py then it is covered by the `import *` (generally import * is frowned upon, but it's okay as long as you define a restrictive `__all__`)